tests/unit/components/utils/costCharts.spec.ts

```diff
@@ -1,35 +1,11 @@
-import type { HSL } from "color-convert";
```

Comment:
Nothing new in this file, except the assertion that costsChartPalette is 3 colors long.